### PR TITLE
[rc] Add extra alternative builds on `aws-sdk-cpp==1.11.210`

### PR DESCRIPTION
**Pull request by jjerphan**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
I have opened #246 and push c99d8c4 as a work-around.

**Review by muhammadhamzasajjad — APPROVED** [review]
LGTM if CI checks🟢

**Comment by jjerphan** [comment]
Hum for some reasons, the test suite of some builds' runs hangs at 90%, it seems.

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -162,7 +161,10 @@ test:
     # For now, we skip the following tests:
     #  - Tests for Azure are deactivated for now (not present for MacOS, fails for Linux).
     #  - `test_lmdb_warnings_when_reopened` also currently fails.
-    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose -n 2 --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl"
+    #  - `test_force_compact_symbol_list_lock_held_past_ttl` also currently fails for MacOS.
+    #  - for `test_write_non_timestamp_index`, see #246
+    #  - for `test_write_not_sorted_multi_index_exception`, see #247
+    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl and not test_write_non_timestamp_index and not test_write_not_sorted_multi_index_exception"
```
Comment (jjerphan) [inline recipe/meta.yaml:167]:
```suggestion
    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl and not test_write_non_timestamp_index and not test_write_not_sorted_multi_index_exception and not test_map_size_bad_input"
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -162,7 +161,10 @@ test:
     # For now, we skip the following tests:
     #  - Tests for Azure are deactivated for now (not present for MacOS, fails for Linux).
     #  - `test_lmdb_warnings_when_reopened` also currently fails.
-    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose -n 2 --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl"
+    #  - `test_force_compact_symbol_list_lock_held_past_ttl` also currently fails for MacOS.
+    #  - for `test_write_non_timestamp_index`, see #246
+    #  - for `test_write_not_sorted_multi_index_exception`, see #247
+    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl and not test_write_non_timestamp_index and not test_write_not_sorted_multi_index_exception and not test_map_size_bad_input"
```
Comment (jjerphan) [inline recipe/meta.yaml:167]:
```suggestion
    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl and not test_write_non_timestamp_index and not test_write_not_sorted_multi_index_exception and not test_map_size_bad_input and not test_map_size_parsing"
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -162,7 +161,10 @@ test:
     # For now, we skip the following tests:
     #  - Tests for Azure are deactivated for now (not present for MacOS, fails for Linux).
     #  - `test_lmdb_warnings_when_reopened` also currently fails.
-    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose -n 2 --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl"
+    #  - `test_force_compact_symbol_list_lock_held_past_ttl` also currently fails for MacOS.
+    #  - for `test_write_non_timestamp_index`, see #246
+    #  - for `test_write_not_sorted_multi_index_exception`, see #247
+    - ARCTICDB_RAND_SEED=$RANDOM pytest python/tests --verbose --reruns 5 --reruns-delay 1 -k "not Azure and not azurite and not test_lmdb_warnings_when_reopened and not test_force_compact_symbol_list_lock_held_past_ttl and not test_write_non_timestamp_index and not test_write_not_sorted_multi_index_exception and not test_map_size_bad_input and not test_map_size_parsing and not lmdb"
```
Comment (jjerphan) [inline recipe/meta.yaml:167]: This might be reverted.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -127,7 +127,6 @@ test:
     # - pyarrow
     - pytest-cpp
     - pytest-timeout
-    - pytest-xdist
```
Comment (jjerphan) [inline recipe/meta.yaml:130]: This might also be reverted.

File: recipe/build.sh [hunk 5]
```diff
@@ -5,9 +5,10 @@ export ARCTICDB_USING_CONDA=1
 # around binary operators' template specialization expansion and compilation.
 # See: https://github.com/man-group/ArcticDB/blob/master/cpp/arcticdb/processing/operation_dispatch_binary.hpp#L451-L473
 #
+# It also has been observed to hang on `version/symbol_list.cpp`.
 # The feedstock's machine have 6.759GiB of RAM and 2 (physical?) cores so we cap it to that.
-# We build with only 2 threads to prevent these freezes from happening.
-export CMAKE_BUILD_PARALLEL_LEVEL=2
+# We build with only 1 thread to prevent these freezes from happening.
+export CMAKE_BUILD_PARALLEL_LEVEL=1
```
Comment (jjerphan) [inline recipe/build.sh:11]: This is necessary for the compilation to pass now.

**Review by muhammadhamzasajjad — APPROVED** [review]

File: recipe/build.sh (hunk 5 above)
Comment (muhammadhamzasajjad) [reply recipe/build.sh:11]: Are you sure that this is causing the build to fail? We have been using multiple threads in ArcticDB CI.

**Pull request merged**